aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Host/Startup/Program.cs

```diff
@@ -12,8 +14,14 @@ public static void Main(string[] args)
 
         public static IWebHost BuildWebHost(string[] args)
         {
+            var config = new ConfigurationBuilder()
+               .SetBasePath(Directory.GetCurrentDirectory())
+               .AddJsonFile("appsettings.json", optional: false)
+               .Build();
+
             return WebHost.CreateDefaultBuilder(args)
                 .UseStartup<Startup>()
+                .UseUrls(config.GetValue<string>("App:ServerRootAddress"))
```

Comment:
hi
Set the url in `appsettings.json` will be clearer.
```json
  "Kestrel": {
    "Endpoints": {
      "Http": {
        "Url": "http://localhost:21021/"
      }
    }
  }
```

Reply:
I can do it. The idea was to use only one webserver config for kestrel and CORS.